### PR TITLE
sort_candidates_by_name

### PR DESCRIPTION
**Pull request by VimalUniyal**
Targets #2952 

### Description
Sorting the candidates in ascending order by their name in Codetrek Page.

- [x] I have performed a self-review of my own code.

### CODE REVIEW

**Comment by rahulpanwar16** [comment]
@VimalUniyal  There are a few suggestions that needs to be done when you are making the PR.

```In Assignees``` : Assign it to yourself before assigning to the reviewers.
```In Label``` : Put ready for review label before assigning it to the reviewers.
```In Reviewers``` : Assign it to team-portal-reviewers.

File: Modules/CodeTrek/Resources/views/index.blade.php [hunk 1]
```diff
@@ -82,6 +82,7 @@ class="nav-link btn-nav {{ request()->input('status') == 'completed' ? 'active'
                         @foreach ($applicants as $applicant)
                             <tr>
                                 <td>
+
```
Comment (rahulpanwar16) [inline Modules/CodeTrek/Resources/views/index.blade.php:85]: Why there a space in between?

File: resources/views/hr/application/index.blade.php [hunk 2]
```diff
@@ -73,7 +73,7 @@
             {{-- Commenting, because we need to brainstorm on this feature a bit --}}
             {{--<div class="mr-2 mt-2 mt-md-0 form-group">
                 <label id="sortby">{!! __('SortBy') !!}</label><br>
-                <select class="fz-14 fz-lg-16 w-120 w-140 form-control rounded border-0 bg-white" name="sort_by" id="sortby"
+                <select class="fz-14 fz-lg-16 w-120 w-140 form-control rounded border-0 bg-white" name="sortby" id="sortby"
```
Comment (rahulpanwar16) [inline resources/views/hr/application/index.blade.php:76]: Why have you done changes on commented code?

**Review by rahulpanwar16 — CHANGES_REQUESTED** [review]
Please check the comments.

**Review by rahulpanwar16 — APPROVED** [review]

**Review by GauravGusain98 — APPROVED** [review]
LGTM

**Pull request merged**